# Motor 1 & 2 encoder values
uint32 m1_enc_val
uint32 m2_enc_val

# Motor 1 & 2 speed values in QPPS (+/-)
int32 m1_enc_qpps
int32 m2_enc_qpps

# Motor 1 & 2 current in Amps
float32 m1_current
float32 m2_current

# Roboclaw controller temperature (C)
float32 temp

# Voltage of the main battery (V)
float32 main_battery_v